package.json

```diff
@@ -60,6 +64,7 @@
     ]
   },
   "generator": {
+    "type": "react",
```

Comment:
Also, I know it's still a draft but I'd consider using another word here. `type` is very generic and can be confusing. What about `renderer`?

Reply:
@jonaslagoni Maybe `renderer`? 😄

Reply:
yes lets go with `renderer` :smile: